components/experimental/src/measure/parser/ids.rs

```diff
@@ -168,3 +168,110 @@ pub const DECIMAL_PREFIXES_TRIE: ZeroTrieSimpleAscii<[u8; 1332]> =
         ("year", 153_usize),
         ("year-person", 154_usize),
     ]);
+
+#[cfg(test)]
+mod tests {
+    use crate::measure::parser::CLDR_IDS_TRIE;
+
+    // This test is to ensure that that the IDs is fixed and not changed.
+    #[test]
```

Comment:
this test is just noise because it repeats code from above. is this linked to anything in datagen, or is this file the source of truth?

Reply:
right now, this file is the source of truth, it is the file the states the ids of each unit

Reply:
A good test would be one that reads the units from CLDR and checks for parity with the hard-coded list in this file. I agree that a test that simply repeats the code above it is not a useful test.

Reply:
Okay, I can also parse the CLDR IDs and ensure that the unit ID is a specific unit ID. However, at the same time, I will need to enumerate all the units to ensure that if any of them change, a test will fail.

Reply:
> I will need to enumerate all the units to ensure that if any of them change, a test will fail.

such "stability tests" are pointless, as explained above.